packages/api/src/providers/oauth1.ts

```diff
@@ -161,23 +161,26 @@ export class OAuth1DefaultImpl implements OAuth1 {
     return { oauth_token, oauth_token_secret };
   }
 
-  async deregister(userAccessTokens: string[]): Promise<void> {
+  async deregister(userAccessTokens: string[], cxId?: string): Promise<void> {
```

Comment:
It'd be nice to add a TSDoc to this function explaining why `cxId` is present and its optional.